### PR TITLE
Any issues encountered during release testing

### PR DESCRIPTION
**Pull request by wasade**
Okay to merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,33 +48,42 @@ If starting from a barcode and read file, you can first use the qiime [split_lib
...
+
+- reference-hit.biom : contains only Deblurred reads matching the positive filtering database. By default, a reference composed of 16S sequences is used, and this resulting table will contain only those reads which recruit at a coarse level to it will be retained. Reads are also filtered against the negative reference, which by default will remove any read which appears to be PhiX or adapter.
+
+- reference-non-hit.biom : contains only Deblurred reads that did not align to the positive filtering database. Negative filtering is also appied to this table, so by default, PhiX and adapter are removed.
+
+- all.biom : contains all Deblurred reads. No filtering against the negative database is performed, meaning that by default, PhiX and adapter sequence will be present in this file. This file is a superset of the union of "reference-hit.biom", "reference-non-hit.biom" and all of the sequences which recruit to the negative reference database which is by default composed of PhiX and adapter.
```
Comment (amnona) [inline README.md]: The all.biom does not contain the sequences aligning to the negative database (so does not contain phix reads). However, no positive filtering is performed (so contains 16s and non-16s sequences)

File: README.md [hunk 2]
```diff
@@ -8,42 +8,35 @@ Deblur is a greedy deconvolution algorithm for amplicon sequencing based on Illu
...
-- reference-hit.biom : contains only deblurred reads matching the positive filtering database (default is only 16S reads)
+If you are running Deblur directly, we recommend focusing on the `workflow` subcommand. Detailed help can be obtained with:
 
-- reference-non-hit.biom : contains only deblurred reads not matching the positive filtering database (default is all non-16S reads)
+```
+deblur workflow --help
+```
 
-The simple use case just specifies the input fasta file (or directory), output directory name and the sequence trim length:
+As a simple example, let's specify an input FASTA file, an output path and a sequence trim length of 150. This command will trim all sequences in `all_samples.fna` to 150nt in length; any read that is shorter will be omitted. This execution mode assumes that `all_samples.fna` is demultiplexed such that the sequence IDs are compatible with QIIME 1.9.1. On completion, a new directory `output` will be created with multiple output files corresponding to the reference databases (see the Input and Output Files section for more detail).
```
Comment (amnona) [inline README.md]: I think stating here "output files corresponding to the reference databases" may confuse a bit the users (as deblur as database independent).
Maybe better to state containing a biom table of the deblurred reads, as well as a biom table of only 16S reads and only non-16S reads (based on positive filtering database - see later)?

File: scripts/deblur [hunk 3]
```diff
@@ -502,16 +512,14 @@ def build_biom_table(seqs_fp, output_biom_fp, min_reads, file_type, log_level,
...
-              help="ATTACK MODE (http://xkcd.com/1692)")
+              help=("indicates this is not the main deblur process "
+                    "(used by the parallel deblur - do not use manually)"))
 def workflow(seqs_fp, output_dir, pos_ref_fp, pos_ref_db_fp,
              neg_ref_fp, neg_ref_db_fp, overwrite,
              mean_error, error_dist, indel_prob, indel_max,
              trim_length, min_reads, min_size, threads_per_sample,
              keep_tmp_files, log_level, log_file, jobs_to_start,
-             is_worker_thread, attack_mode):
+             is_worker_thread):
```
Comment (amnona) [inline scripts/deblur]: I miss attack mode :( - how about keeping it?

**Comment by amnona** [comment]
cool. much more clear.
few minor comments

File: README.md (hunk 2 above)
Comment (wasade) [inline README.md]: remember: deblur is _not_ 16s specific. there also does not appear to be a mode of operation that is database independent. i will revise the text for clarity.

File: README.md (hunk 1 above)
Comment (wasade) [inline README.md]: if `all.biom` is the union of `reference-hits.biom` and `reference-non-hits.biom`, then what is its use? this is just data replication.

File: scripts/deblur (hunk 3 above)
Comment (wasade) [inline scripts/deblur]: no.

**Comment by wasade** [comment]
@amnona @antgonza @josenavas @mortonjt if this is good to merge, can it be merged please?

**Pull request merged**